src/opti_key.yaml

```diff
@@ -0,0 +1,40 @@
+---
+description: >
+  OptiKey is an on-screen keyboard that is designed to help patients with
+  Motor Neuron Disease (MND) interact with Windows computers.
+  It can be used with eye-tracking devices, mouse or webcam.
+
+  At the current time it has also been translated into
+  [24 different
+  languages](https://github.com/OptiKey/OptiKey/wiki/Supported-languages)
+license: GPL-3.0
+name: OptiKey
+programmingLanguages:
+  - c#
+rating: 4
+repoUrl: "https://github.com/OptiKey/OptiKey"
+sdgs: [ 10 ] 
+websiteUrl: "http://www.optikey.org"
+
+contributionGuidelinesUrl: "https://github.com/OptiKey/OptiKey"
+logoUrl: "https://raw.githubusercontent.com/OptiKey/OptiKey/gh-pages/images/\
+          OptiKey-logo-600x200.png"
+starsUrl: "https://img.shields.io/github/stars/OptiKey/OptiKey.svg\
+           ?style=social&label=Star&maxAge=2592000"
+numberContributors:
+  url: "https://api.github.com/repos/OptiKey/OptiKey/stats/contributors"
+  format: json  # or yaml or xml
+  accessor: list  # or number or property name prepended by ?
+
+
+ratingComment: Very active and impactful, but no good Contribution guide
```

Comment:
Maybe we can contact them to improve it? If the project is very active, then maybe the contribution guide isn't so important in their case? We just want to verify that a project is easy to join in the end :)

Reply:
hmm, I'll look for issues addressing this and might open one

Reply:
Oh apparently they estimate to have around 2000 active users as of march. That might be a good info to include…
source: https://github.com/OptiKey/OptiKey/issues/544#issuecomment-469251533

Reply:
Actually I just found https://github.com/OptiKey/OptiKey/wiki/Contribute through googling.
I might still suggest moving that link to the CONTRIBUTING.md at least

Reply:
Nice catch

Reply:
@AblingerOscar @Fandekasp I have added a link from the Contributing.md file to the contributing guide. I will look to enhance this guide soon, but first the solution architecture is being overhauled a bit. OptiKey has grown so much that there is now scope to split it into 4 distinct applications (all under the OptiKey umbrella and part of the same solution) which are tailored to the end user's requirements:

Pro - speech, mouse and keyboard control
Chat - full screen speech
Symbol - communication using symbol boards for pre-literate users
Mouse - only mouse control

Reply:
@JuliusSweetland thank you for your elaboration :)

I think I'll update the contribution link to go to the `CONTRIBUTING.md` file again. Once the project is split up, we'd presumably also have to update some links, but for now, I think that the rest is okay.